packages/x-charts/src/ChartsLabel/ChartsLabelMark.tsx

```diff
@@ -34,6 +40,8 @@ const Root = styled('div', {
     alignItems: 'center',
     justifyContent: 'center',
     flexShrink: 0,
+    width: 14,
+    height: 14,
```

Comment:
Had to define a default size here so that the custom component defaults to occupying 100% of the width/height.

Reply:
You would need to set the height on `labelMarkClasses.line` as `unset`. It added a few pixels to the demos, hence all the changes 🫠